### DISCUSSION TITLE
Removal of `case sensitive` config option

### DISCUSSION THREAD

**Discussion by APickledWalrus** [SkriptLang/Skript#4823] [Suggestions]
I am proposing that we remove the following config option:
https://github.com/SkriptLang/Skript/blob/0c74e07bdd9ae12cce9c6f1de716608bc72b05b3/src/main/resources/config.sk#L139-L142

Instead, each syntax making use of this config option should have an individual `with case sensitivity` option that can be specified. This can already be found in EffReplace: https://github.com/SkriptLang/Skript/blob/0c74e07bdd9ae12cce9c6f1de716608bc72b05b3/src/main/java/ch/njol/skript/effects/EffReplace.java#L57-L60

Why? In the same way that we removed non-message stuff from non-English localization files (as we decided that non-message stuff shouldn't be translated, since it could break script compatibility between languages), I think that this setting only serves to cause a similar problem. It would also be far more useful for users to be able to switch between case sensitivity checks, rather than be forced to go all in for all syntaxes.

**Comment by TheLimeGlass** [comment]
Doesn't this setting mainly impact variable names? I don't mind the syntax addition regardless of the removal.
I think this impacts variables too much, but I don't see people using case and non case same variables.

Add option to the bStats to see how many actually use this option https://bstats.org/plugin/bukkit/Skript/722
